test/framework/fixtures/libgoalFixture.go

```diff
@@ -499,3 +502,50 @@ func (f *LibGoalFixture) MinFeeAndBalance(round uint64) (minFee, minBalance uint
 	}
 	return params.MinTxnFee, minBalance, nil
 }
+
+// ProofRespToProof returns a proof for usage in merkle array verification from the inputted API proof query response.
+func (f *LibGoalFixture) ProofRespToProof(proofResp generatedV2.ProofResponse) (proof merklearray.Proof, err error) {
```

Comment:
maybe you should not use names for return arguments

Reply:
done